src/frontend/packages/core/src/features/cloud-foundry/users/remove-user/remove-user.component.ts

```diff
@@ -0,0 +1,179 @@
+import { Component, OnDestroy } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { Store } from '@ngrx/store';
+import { Observable, of as observableOf } from 'rxjs';
+import { combineLatest, first, map } from 'rxjs/operators';
+
+import {
+  UsersRolesClear,
+  UsersRolesExecuteChanges,
+  UsersRolesSetChanges,
+  UsersRolesSetUsers,
+} from '../../../../../../store/src/actions/users-roles.actions';
+import { AppState } from '../../../../../../store/src/app-state';
+import { selectUsersRoles } from '../../../../../../store/src/selectors/users-roles.selector';
+import { CfUser, IUserPermissionInOrg, IUserPermissionInSpace } from '../../../../../../store/src/types/user.types';
+import { StepOnNextFunction } from '../../../../shared/components/stepper/step/step.component';
+import { CfUserService } from '../../../../shared/data-services/cf-user.service';
+import { ActiveRouteCfOrgSpace } from '../../cf-page.types';
+import { getActiveRouteCfOrgSpaceProvider } from '../../cf.helpers';
+import { CfRolesService } from '../manage-users/cf-roles.service';
+
+
+@Component({
+  selector: 'app-remove-user',
+  templateUrl: './remove-user.component.html',
+  styleUrls: ['./remove-user.component.scss'],
+  providers: [
+    getActiveRouteCfOrgSpaceProvider,
+    CfUserService,
+    CfRolesService
+  ]
+})
+export class RemoveUserComponent implements OnDestroy {
+  initialUsers$: Observable<CfUser[]>;
+  singleUser$: Observable<CfUser>;
+  defaultCancelUrl: string;
+  orgGuid: string;
+  spaceGuid: string;
+  applyStarted = false;
+  onlySpaces = false;
+
+  constructor(
+    private store: Store<AppState>,
+    private activeRouteCfOrgSpace: ActiveRouteCfOrgSpace,
+    private cfUserService: CfUserService,
+    private cfRolesService: CfRolesService,
+    private route: ActivatedRoute
+  ) {
+    this.defaultCancelUrl = this.createReturnUrl(activeRouteCfOrgSpace);
+    this.orgGuid = this.activeRouteCfOrgSpace.orgGuid;
+    this.spaceGuid = this.activeRouteCfOrgSpace.spaceGuid;
+    this.onlySpaces = this.route.snapshot.queryParamMap.get('spaces') === 'true';
+
+    const userQParam = this.route.snapshot.queryParamMap.get('user');
+    if (userQParam) {
+      this.singleUser$ = this.cfUserService.getUser(activeRouteCfOrgSpace.cfGuid, userQParam)
+        .pipe(
+          map(user => user.entity),
+        );
+      // catchError and throw 404
+    } else {
+      console.log('user not found page');
```

Comment:
Something would have gone quite wrong to get here, but i think just a logger.error using the LoggerService service and a `return` will do